atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,9 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        media_types = ['application/json']
```

Comment:
We currently don't run a configuration in which we do not produce v1 content only (pulp_push always runs). But that won't always be the case. Hard coding it like this adds an incorrect dependency IMO.